OutOfSchool/OutOfSchool.WebApi/Extensions/ApiErrorExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Ardalis.SmartEnum;
+using OutOfSchool.WebApi.Common.Responses;
+
+namespace OutOfSchool.WebApi.Extensions;
+
+public static class ApiErrorExtensions
+{
+    public static ApiErrorResponse CreateApiErrorResponse(this ApiError apiError)
```

Comment:
(Code style) Consider necessity of this extension class.
Ususally extensions are good for code decoupling. Mapping extensions - when you have 2 classses not related to each other and you don't want introduce any hard dependency between them. 

It is not the case for `ApiError` and `ApiErrorResponse`, as latter is direct child of former. So why not put this method in `ApiErrorResponse` class?